config/config.yaml.example

```diff
@@ -42,3 +42,12 @@ resources:
         mem_mb: 64000
         threads: 24
         params_extra: "/path/to/clair3"
+    smoothcov:
+        partition: short
+        threads: 8
+        mem_mb: 18000
+        runtime: 30
+        bin: 1000000
+        chunk: 500000
+        spike: 100
+        smooth: 3
```

Comment:
These lines are parameters for the software itself, not for the resources in the cluster, so they should be defined elsewhere.